index.html

```diff
@@ -0,0 +1,111 @@
+      <button id="menu-icon" class="toggle"><i class="fa-solid fa-bars"></i></button>
+      <nav class="navbar">
+        <a href="#home">Web School</a>
+        <div class="navbar-container">
+          <ul class="navbar-list">
+          <li class="navbar-item"><a href="#about">About </a></li>
+          <li class="navbar-item"><a href="#about">Program</a></li>
+          <li class="navbar-item"><a href="#home">Join</a></li>
+          <li class="navbar-item"><a href="#Scholarship">Scholarship</a></li>
+          <li class="navbar-item"><a href="#home">Upcomig classes</a></li>
+        </ul>
+        <button class="sign-up"><a href="#">SignUp</a></button>
+        </div>
+      </nav>
+      <div class="home-body" id="home">
+        <p class="body-disc">"Hello! Future Web Developers"</p>
+        <h1 class="home-heading">WE ARE HERE TO TRAIN THE BEST WEBSITE<span> DEVELOPERS FOR THE MARKET</span> IN 2023.</h1>
+        <p class="body-disc2">We provide the best suitable teaching method that can turn anyone to be the best in the market. And we also provide a flexable study time.</p>
+        <h2 class="heading2">Registration for 2023 is Open<span> (Feb16) ~ (Feb21)</span></h2>
+      </div>
+    </header>
+  </section>  
+
+  <section class="about" id="about">
+    <h2 class="about-heading">Our Teaching Methods</h2>
+    <div class="heading-under"></div>
+    <div class="about-list">
+      <div class="about-item">
+        <i class="fa-solid fa-person-chalkboard"></i>
+        <h3 class="about-item-title">Lecture</h3>
+        <p class="about-item-disc">We provide a live lecture classes by the best teachers in the topic.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-bars-progress"></i>
+        <h3 class="about-item-title">Projects</h3>
+        <p class="about-item-disc">During the program there are lots of real world projets for practical purposes.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-user-group"></i>
+        <h3 class="about-item-title">Pair Coding</h3>
+        <p class="about-item-disc">We will pair you with another student to stay motivated and also to learn from each other.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-code"></i>
+        <h3 class="about-item-title">Code reviews</h3>
+        <p class="about-item-disc">Every project you submit will be reviewed senior developers inorder to make you sharp.</p>
+      </div>
+      <div class="about-item">
+        <i class="fa-solid fa-people-pulling"></i>
+        <h3 class="about-item-title">Mentor</h3>
+        <p class="about-item-disc">In Every steps you are about to take we will make sure you get the most appropriate consling.</p>
+      </div>
+    </div>
+    <button class="about-btn">Register Now</button>
+    <div class="about-link">
+      <a href="#">See The Whole Program</a>
+    </div>
+  </section>
+
+  <div class="section-feature">
+
+  </div>
```

Comment:
There is an important section of your project missing which is the about page please add this page and try again

![image](https://user-images.githubusercontent.com/60612329/219875501-b41efdae-8a23-47f2-b76d-bb6fc44c4d12.png)